DESCRIPTION

```diff
@@ -34,6 +32,7 @@ Imports:
     stringr,
     tibble,
     tidyselect,
+    vthemes,
```

Comment:
Without `vthemes`, the automated rmd (using `create_rmd()`) does not work because it depends on `vthemes::subchunkify()` which is a really useful rmd utility function. For example, `vthemes::subchunkify()` allows us to create multiple plots of different sizes within the same R chunk. This is very useful within for loops. Given this, do you think it's still ok to put `vthemes` in Suggests?

Reply:
I completely agree though that we should try to reduce the number of imports and remove some packages like `progressr`. I'll create an issue for this. #

Reply:
> Without `vthemes`, the automated rmd (using `create_rmd()`) does not work because it depends on `vthemes::subchunkify()` which is a really useful rmd utility function. For example, `vthemes::subchunkify()` allows us to create multiple plots of different sizes within the same R chunk. This is very useful within for loops. Given this, do you think it's still ok to put `vthemes` in Suggests?

Nice, that is super useful! Ya, I think the same thing applies. If the user calls create_rmd(), rlang::check_installed() will ask them to install vthemes. Ideally, at some point in the future `create_doc_template()` and `create_rmd()` will just be a simple wrappers around functionality in `vdocs` and `vthemes`. 

I think the rationale behind using suggests rather than imports as done in `purrr` is that `dplyr` has way more functionality than `purrr` needs, and `purrr` only uses it in a small number of its functions. Right now I don't think that's the case with `vthemes` relative to `simChef`, but as `vthemes` grows it might become so. Let's leave it in imports, but just something to keep in mind as things develop.